contents/ui/main.qml

```diff
@@ -126,6 +126,7 @@ Item {
     readonly property Item activeTaskItem: windowInfoLoader.item.activeTaskItem
 
     property string fallBackText: {
+        if(fullActivityInfo.name === "Default") { return "    "; } // WHITESPACE SO USER CAN STILL ACCESS CONFIGURATION
```

Comment:
this change is not acceptable...
"Default" is chosen by Plasma Devs so changing it or removing it is no solution...